packages/graphile-build-pg/src/plugins/PgColumnsPlugin.js

```diff
@@ -151,6 +150,7 @@ export default (function PgColumnsPlugin(
         fields,
         introspectionResultsByKind.attribute
           .filter(attr => attr.classId === table.id)
+          .filter(attr => pgColumnFilter(attr, build, Object.assign({}, context, {type: "GraphQLObjectType"})))
```

Comment:
I can't expand on mobile but is this one meant to be GraphQLInputObjectType?

Reply:
correct, it’s fixed